src/gui/qgstextformatwidget.cpp

```diff
@@ -132,20 +132,20 @@ void QgsTextFormatWidget::initWidget()
 
   // color buttons
   mPreviewBackgroundBtn->setColorDialogTitle( tr( "Select fill color" ) );
-  mPreviewBackgroundBtn->setContext( QStringLiteral( "labelling" ) );
+  mPreviewBackgroundBtn->setContext( QStringLiteral( "labeling" ) );
```

Comment:
Not really. It's a feature of the color buttons that they can tweak the list of displayed colors in the drop down menu by context. But it's not used anywhere in master and afaik there's no plugins which add color schemes. It's safe to change.